### PR TITLE
Add Belarusian Language Pack

### PR DESCRIPTION
**Pull request by anibyl**
This PR adds Belarusian Language Pack.
The dictionary might need tweaking in the future.
Issue #2363.

### CODE REVIEW

File: addons/languages/belarusian/apk/build.gradle [hunk 1]
```diff
@@ -0,0 +1,7 @@
+ext.shouldBePublished = true
```
[the remark this answers is not in the document]
Comment (anibyl) [reply addons/languages/belarusian/apk/build.gradle:1]: Got it. Let me know when you finish reviewing the PR, I'll push all the changes at the same time. The checks take plenty of time.

[the remark this answers is not in the document]
Comment (anibyl) [reply addons/languages/belarusian/apk/build.gradle]: Also, there is a pack published under your account in 2010. https://play.google.com/store/apps/details?id=com.anysoftkeyboard.languagepack.belarusian

**Comment by anibyl** [comment]
@menny could you please take a look at this PR once again?

**Comment by menny** [comment]
thank you

**Pull request merged**